packages/http-client-csharp/generator/TestProjects/Local/Unbranded-TypeSpec/src/Generated/UnbrandedTypeSpecClient.RestClient.cs

```diff
@@ -56,12 +56,12 @@ internal PipelineMessage CreatePatchActionRequest(BinaryContent content, Request
             throw new NotImplementedException("Method not implemented.");
         }
 
-        internal PipelineMessage CreateAnonymousBodyRequest(BinaryContent content, RequestOptions options)
+        internal PipelineMessage CreateAnonymousBodyRequest(RequestOptions options)
```

Comment:
spread of model/alias is not properly implemented yet here, therefore we see this change.